static/edx-pl/sass/main-ltr.scss

```diff
@@ -0,0 +1,22 @@
+// ------------------------------
```

Comment:
@ormsbee and @feanil, **quick gut check needed**: I've moved a static version of the edX Pattern Library (https://github.com/edx/ux-pattern-library) here for web certs to use now (and down the road future things to leverage). I know we need to cover packaging for easier maintenance and versioning, but for now:

* is this alright by you?
* is this the right location (a global static directory) for this set of UI architecture?